contrib/buildrpm.sh

```diff
@@ -117,6 +117,7 @@ if [ $opt_binrpm -eq 1 ]; then
 	with_args+=" $(with_arg mad)"
 	with_args+=" $(with_arg mlx5)"
 	with_args+=" $(with_arg efa)"
+	with_args+=" $(with_arg gdaki)"
```

Comment:
probably thr module name "gdaki" should be more specific to capture which network device and GPU are used. to allow adding more "gdaki" transport for other GPU and network 
Possible names:
"ibgda"
"mlx5-gdaki"
"mlx5-gda"